.github/ci/sanitizer/clang/Leak.supp

```diff
@@ -11,3 +11,10 @@ leak:/usr/bin/make*
 # ADIOS 1.13.1
 leak:adios_read_bp_open_file
 leak:adios_inq_var
+# ADIOS2
```

Comment:
Are those still up-to-date?
@chuckatkins et al. added ASan/UBSan checks to ADIOS2 end of last year.

Reply:
Just tried it out, seems I can throw them out.

Reply:
Well, it seems [that](https://travis-ci.com/github/openPMD/openPMD-api/jobs/431253555) answers that.
```
==10270==ERROR: LeakSanitizer: detected memory leaks

Direct leak of 176 byte(s) in 1 object(s) allocated from:

    #0 0x7fbdfe615043 in __interceptor_malloc /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_malloc_linux.cc:146:3

    #1 0x18445d6 in load_transport (/home/travis/build/bin/ParallelIOTests+0x18445d6)

    #2 0x18379e2 in INT_CMlisten_specific (/home/travis/build/bin/ParallelIOTests+0x18379e2)

    #3 0x180ed32 in CMlisten (/home/travis/build/bin/ParallelIOTests+0x180ed32)

    #4 0x17d31c6 in CP_getCPInfo (/home/travis/build/bin/ParallelIOTests+0x17d31c6)

    #5 0x17cbed7 in SstReaderOpen (/home/travis/build/bin/ParallelIOTests+0x17cbed7)

    #6 0x17731ae in adios2::core::engine::SstReader::SstReader(adios2::core::IO&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, adios2::Mode, adios2::helper::Comm) (/home/travis/build/bin/ParallelIOTests+0x17731ae)

    #7 0x147fdc5 in std::shared_ptr<adios2::core::Engine> adios2::core::IO::MakeEngine<adios2::core::engine::SstReader>(adios2::core::IO&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, adios2::Mode, adios2::helper::Comm) (/home/travis/build/bin/ParallelIOTests+0x147fdc5)

    #8 0x13eb5d8 in std::_Function_handler<std::shared_ptr<adios2::core::Engine> (adios2::core::IO&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, adios2::Mode, adios2::helper::Comm), std::shared_ptr<adios2::core::Engine> (*)(adios2::core::IO&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, adios2::Mode, adios2::helper::Comm)>::_M_invoke(std::_Any_data const&, adios2::core::IO&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, adios2::Mode&&, adios2::helper::Comm&&) (/home/travis/build/bin/ParallelIOTests+0x13eb5d8)
```

Reply:
```
leak:adios2::core::engine::SstReader::*
leak:adios2::core::engine::BP4Reader::*
leak:adios2::core::engine::SstWriter::*
leak:adios2::core::engine::BP4Writer::*
leak:CMpoll_forever
leak:SstWriterClose
```
Of those, the `SstWriter` and `SstReader` lines are still needed, the other ones not any longer.